src/test/java/com/google/api/generator/gapic/composer/BUILD.bazel

```diff
@@ -38,6 +39,7 @@ TEST_DEPS = [
     "//src/main/java/com/google/api/generator/gapic/protoparser",
     "//src/test/java/com/google/api/generator/gapic/testdata:showcase_java_proto",
     "//src/test/java/com/google/api/generator/gapic/testdata:testgapic_java_proto",
+    "@com_google_api_api_common//jar",
```

Comment:
What is this new dep used for?

Reply:
`ConcreteReference.withClazz(com.google.api.resourcenames.ResourceName.class)`
This is used in unit tests. To create resource name type.

Reply:
Thanks for clarifying.